wordpress-mobile/WordPress-Android
# Stats Refresh: Follower totals block

Fixes #9460.

This PR adds a new Follower totals block.

![image](https://user-images.githubusercontent.com/1522856/56214594-78b79400-605e-11e9-9420-c7bed28c52be.png)

To test:

1. Go to Stats
2. Tap on the Insights tab
3. Scroll down and notice there is a new Follower Totals block

Update release notes:
